osi_environment.proto

```diff
@@ -119,18 +128,31 @@ message EnvironmentalConditions
         // Other (unspecified but known) ambient illumination.
         //
         AMBIENT_ILLUMINATION_OTHER = 1;
-        // Ambient illumination index is "bright".
-        // E.g. daytime with sunshine.
-        AMBIENT_ILLUMINATION_BRIGHT = 2;
-        // Ambient illumination index is "medium".
-        // E.g. daytime with overcast sky.
-        AMBIENT_ILLUMINATION_MEDIUM = 3;
-        // Ambient illumination index is "low".
-        // E.g. during twilight (dawn/dusk) or in a tunnel.
-        AMBIENT_ILLUMINATION_LOW = 4;
-        // Ambient illumination index is "dark".
-        // E.g. night time, no sunlight.
-        AMBIENT_ILLUMINATION_DARK = 5;
+        // Illumination between [0.001, 0.01[ Lux is Level 1.
+        // E.g. Night with no artificial light.
+        AMBIENT_ILLUMINATION_LEVEL1 = 2;
+        // Illumination between [0.01, 1[ Lux is Level 2.
+        // E.g. Night full moon / Deep twilight.
+        AMBIENT_ILLUMINATION_LEVEL2 = 3;
+        // Illumination between [1, 3[ Lux is Level 3.
+        // E.g. Deep to average twilight / Minimum lighting on local low pedestrian conflict roads.
+        AMBIENT_ILLUMINATION_LEVEL3 = 4;
+        // Illumination between [3, 10[ Lux is Level 4.
+        // E.g. Average to full twilight / Minimum lighting on collector roads /
+        // Minimum lighting on major and expressway roads with low to average pedestrian conflict.
+        AMBIENT_ILLUMINATION_LEVEL4 = 5;
+        // Illumination between [10, 20[ Lux is Level 5.
+        // E.g. Minimum lighting on major and expressway roads with high pedestrian conflict.
+        AMBIENT_ILLUMINATION_LEVEL5 = 6;
+        // Illumination between [20, 1000[ Lux is Level 6.
+        // E.g. Roads with more lighting / Dark or overcast day.
+        AMBIENT_ILLUMINATION_LEVEL6 = 7;
```

Comment:
We would suggest to include another level here. The 400 lux limit would correspond to a sunrise on a clear day which might be relevant for a camera model.

Reply:
I can split this category into a [20, 400] lux level and a [400 to 1000] lux level.